### PR TITLE
DropDownButton: Create TestCafe test for DropDownButton Overview

### PR DESCRIPTION
**Pull request by jdvictoria**

### CODE REVIEW

**Comment by jdvictoria** [comment]
[WG Tests - DropDownButton](https://trello.com/c/clVMuJvX/8598-wg-tests-dropdownbutton)

File: apps/demos/testing/widgets/dropdownbutton/DropDownButton.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+fixture('DropDownButton.Overview')
+  .page('http://localhost:8080/')
+  .beforeEach(async (t) => {
+    await t.resizeWindow(900, 800);
+  });
+
+runManualTest('DropDownButton', 'Overview', ['jQuery', 'React', 'Vue', 'Angular'], (test) => {
+  test('Custom Overview Appearance', async (t) => {
+    const STATIC_BUTTON_CLASS = 'dx-dropdownbutton';
```
Comment (marker-dao) [inline apps/demos/testing/widgets/dropdownbutton/DropDownButton.test.js]: Let's move variables to top, after imports.

File: apps/demos/testing/widgets/dropdownbutton/DropDownButton.test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector as $ } from 'testcafe';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+
+fixture('DropDownButton.Overview')
+  .page('http://localhost:8080/')
+  .beforeEach(async (t) => {
```
Comment (EugeniyKiyashko) [inline apps/demos/testing/widgets/dropdownbutton/DropDownButton.test.js]: The same as here: https://github.com/DevExpress/DevExtreme/pull/26891/files#r1521260396

File: apps/demos/testing/widgets/dropdownbutton/DropDownButton.test.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector as $ } from 'testcafe';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+
+const STATIC_BUTTON_CLASS = 'dx-dropdownbutton';
+const CUSTOM_BUTTON_SELECTOR = 'dx-dropdownbutton-toggle';
```
Comment (EugeniyKiyashko) [inline apps/demos/testing/widgets/dropdownbutton/DropDownButton.test.js]:
```suggestion
const DROP_DOWN_BUTTON_TOGGLE_CLASS = 'dx-dropdownbutton-toggle';
```

File: apps/demos/testing/widgets/dropdownbutton/DropDownButton.test.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector as $ } from 'testcafe';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+
+const STATIC_BUTTON_CLASS = 'dx-dropdownbutton';
```
Comment (EugeniyKiyashko) [inline apps/demos/testing/widgets/dropdownbutton/DropDownButton.test.js]:
```suggestion
const DROP_DOWN_BUTTON_CLASS = 'dx-dropdownbutton';
```

**Review by ksercs — DISMISSED** [review]

File: apps/demos/testing/widgets/dropdownbutton/DropDownButton.test.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector as $ } from 'testcafe';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+
+const DROP_DOWN_BUTTON_TOGGLE_CLASS = 'dx-dropdownbutton';
+const DROP_DOWN_BUTTON_CLASS = 'dx-dropdownbutton-toggle';
```
Comment (ksercs) [inline apps/demos/testing/widgets/dropdownbutton/DropDownButton.test.js]: classes seem to be reversed

**Review by ksercs — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**